netbox/utilities/management/commands/calculate_cached_counts.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from django.core.management.color import no_style
+
+from dcim.models import Device
+from virtualization.models import VirtualMachine
+
+
+def recalculate_device_counts():
```

Comment:
Ideally we'd have some mechanism to discover all the necessary fields, rather than defining them explicitly. Again, this is where utilizing the registry can help.